dashboard/test/models/census/census_teacher_banner_v1_test.rb

```diff
@@ -0,0 +1,23 @@
+require 'test_helper'
+
+class Census::CensusTeacherBannerV1Test < ActiveSupport::TestCase
+  test "basic census teacher banner v1 submission is valid" do
+    submission = build(:census_teacher_banner_v1)
+    assert submission.valid?, submission.errors.full_messages
+  end
+
+  test "teacher banner v1 submission for non-teacher is invalid" do
+    submission = build(:census_teacher_banner_v1, submitter_role: "PARENT")
+    refute submission.valid?, submission.errors.full_messages
```

Comment:
Why is `submission.errors.full_messages` the message here? If the `refute` fails, then the submission will be valid and that will be an empty array.

Reply:
Same below. I'd just remove the custom message for the 2 refutes. It makes sense for the assert on line 21.